interfaces/console/src/app/console/sites/page.tsx

```diff
@@ -297,139 +126,58 @@ export default function Page() {
     }
   };
 
-  const handleBatteryStatSubscription = (_: any, data: string) => {
-    try {
-      const parsedData = JSON.parse(data);
-      const { value, type, success, siteId } = parsedData.data.getMetricStatSub;
-
-      if (success && type === 'battery_charge_percentage') {
-        setSitesStatus((prev) => {
-          const currentStatus = prev[siteId] || {
-            status: SITE_STATUS.ONLINE,
-            batteryStatus: 'Charged',
-            signalStrength: 'Strong',
-          };
-
-          let batteryStatus = 'Charged';
-          let newStatus = currentStatus.status;
-
-          if (value[1] < 20) {
-            batteryStatus = 'Low';
-            if (newStatus !== SITE_STATUS.OFFLINE) {
-              newStatus = SITE_STATUS.WARNING;
-            }
-          } else if (value[1] < 50) {
-            batteryStatus = 'Medium';
-          }
-
-          return {
-            ...prev,
-            [siteId]: {
-              ...currentStatus,
-              batteryStatus,
-              status: newStatus,
+  const fetchSiteMetrics = async () => {
+    const to = getUnixTime();
+    const from = to - 30;
+
+    for (const site of sitesList) {
+      try {
+        const res = await getSiteStatsMetrics({
+          variables: {
+            data: {
+              siteId: site.id,
+              type: Stats_Type.Site,
+              from,
+              to,
+              step: 30,
+              userId: user.id,
+              orgName: user.orgName,
+              withSubscription: true,
             },
-          };
+          },
```

Comment:
Doesn't look right. This should be manage at BFF level. For console it should be just a type. i.e: `ALL_SITES_HEALTH_KPIS`